console/package.json

```diff
@@ -101,7 +103,9 @@
     "redux-thunk": "^2.2.0",
     "sanitize-filename": "^1.6.1",
     "semver": "5.5.1",
+    "source-map-loader": "^0.2.4",
```

Comment:
Can't these go into dev dependencies? They won't be needed in the prod bundle right?
Just curious.

Reply:
It can. Will update this.

Reply:
Hey! All the typescript related dependencies I meant. We don't need them in production right?

Reply:
Updated!